submissions/kirill8210/oop-task/app.js

```diff
@@ -0,0 +1,105 @@
+const fieldX = 505;
+const playerX = 205;
+const playerY = 405;
+const playerHight = 60;
+const playerWidth = 80;
+const playerMoveX  = 102;
+const playerMoveY  = 83;
+
+const Enemy = function(x, y, speed) {
+    this.x = x;
+    this.y = y;
+    this.speed = speed;
+    this.sprite = 'images/enemy-bug.png';
+};
+
+Enemy.prototype.update = function (dt) {
+    this.x += this.speed * dt;
+
+    if (this.x > fieldX) {
+        this.x = -50;
+        this.speed = 50 + Math.floor(Math.random() * 200);
+    }
+
+    if (player.x < this.x + playerWidth &&
+        player.x + playerWidth > this.x &&
+        player.y < this.y + playerHight &&
+        playerHight + player.y > this.y) {
+        player.x = playerX;
+        player.y = playerY;
```

Comment:
See item 4.iii of the [requirements](https://kottans.org/documentation/docs/doc/code-review/#oo-js-frogger) linked by bot.

Reply:
fixed this error